File: Easy/River I/RIVER I.cs
using System;
using System.Linq;
using System.IO;
using System.Text;
using System.Collections;
using System.Collections.Generic;

class Solution
{
    static void Main(string[] args)
    {
        int r1value = int.Parse(Console.ReadLine());
        int r2value = int.Parse(Console.ReadLine());


        while (r1value != r2value)
        {
            if (r1value > r2value)
                r2value = r2value + getnextnumber(r2value);
            else if (r2value > r1value)
                r1value = r1value + getnextnumber(r1value);
        }
        Console.WriteLine(r1value);
    }

    static int getnextnumber(int input)
    {
        int result = 0;
        for ( int i = 0; i < Convert.ToString(input).Length; i++)
        {
            result += int.Parse(Convert.ToString(input).Substring(i,1));
        }
        return result;
    }
}

File: Easy/River I/RIVER I.py
import sys
import math

def getnextnumber(i):
    result = 0
    for element in str(i):
        result += int(element)
    return result

r_1 = int(input())
r_2 = int(input())

while r_1 != r_2:
    if(r_1>r_2):
        r_2 += getnextnumber(r_2)
    elif(r_1<r_2):    
        r_1 += getnextnumber(r_1)

print(r_1)